test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -91,5 +80,19 @@ public class LoggingTests : IISFunctionalTestBase
             }
         }
 
+        private string GetFileContents(string file)
+        {
+            Assert.NotNull(file);
+
+            string contents = null;
+
+            // RetryOperation doesn't support async lambdas, call synchronous ReadAllText.
+            RetryHelper.RetryOperation(
```

Comment:
You don't need this retry. Server should be stopped and file accessible at the point where we try to read it.